gui-apps/wf-recorder/wf-recorder-9999.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="Screen recorder for wlroots-based compositors"
+HOMEPAGE="https://github.com/ammen99/wf-recorder"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/ammen99/wf-recorder.git"
+else
+	SRC_URI="https://github.com/ammen99/wf-recorder/releases/download/v${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="+man opencl"
+
+DEPEND="
+	dev-libs/wayland
+	media-sound/pulseaudio
```

Comment:
Checking upstream, pulseaudio support seems to be optional now. Please update -9999 ebuild to reflect that.
https://github.com/ammen99/wf-recorder/commit/a9c0cc14f6dad3e9067db1fbd85f4275c6b0b50e

Reply:
done